packages/expo-maps/android/build.gradle

```diff
@@ -1,41 +1,18 @@
-apply plugin: 'com.android.library'
-
-def useLegacyExpoModulesCorePlugin = {
-  def expoModulesCorePlugin = new File(project(":expo-modules-core").projectDir.absolutePath, "ExpoModulesCorePlugin.gradle")
-  apply from: expoModulesCorePlugin
-  applyKotlinExpoModulesCorePlugin()
-  useCoreDependencies()
-  useDefaultAndroidSdkVersions()
-  useExpoPublishing()
-}
-
-try {
-  apply plugin: 'expo-module-gradle-plugin'
-} catch (e) {
-  if (!e instanceof UnknownPluginException) {
-    throw e
-  }
-
-  useLegacyExpoModulesCorePlugin()
-}
-
 buildscript {
-  ext.KOTLIN_MAJOR_VERSION = kotlinVersion.split("\\.")[0].toInteger()
 
-  if (KOTLIN_MAJOR_VERSION >= 2) {
-    repositories {
-      mavenCentral()
-    }
-    dependencies {
-      classpath("org.jetbrains.kotlin.plugin.compose:org.jetbrains.kotlin.plugin.compose.gradle.plugin:${kotlinVersion}")
-    }
+  repositories {
+    mavenCentral()
+  }
+  dependencies {
+    classpath("org.jetbrains.kotlin.plugin.compose:org.jetbrains.kotlin.plugin.compose.gradle.plugin:${kotlinVersion}")
   }
-}
 
-if (KOTLIN_MAJOR_VERSION >= 2) {
-  apply plugin: 'org.jetbrains.kotlin.plugin.compose'
 }
 
+apply plugin: 'com.android.library'
+apply plugin: 'expo-module-gradle-plugin'
+apply plugin: 'org.jetbrains.kotlin.plugin.compose'
+
```

Comment:
there were some modules still using `apply plugins`, e.g. expo-maps and expo-ui. is that intentional?

Reply:
Yes, it's because otherwise `org.jetbrains.kotlin.plugin.compose` won't be available. This should be included in the plugin manager block in `settings.gradle`, and I plan to do this in a separate PR.